### ISSUE TITLE
gammu-smsd loops without receiving or sending Checking line: +ZEND

### ISSUE THREAD

**Issue by marcosandrade22** [gammu/gammu#220]
Good afternoon, gammu works well for me, but at some point it enters a loop and I can no longer send or receive messages , follows the log

```gammu: 1 "+ZEND"
Mon 2016/06/27 14:48:41 gammu-smsd[3185]: gammu: Checking line: +ZEND

Mon 2016/06/27 14:48:41 gammu-smsd[3185]: gammu: AT reply state: 4

Mon 2016/06/27 14:48:41 gammu-smsd[3185]: gammu: RECEIVED frame type 0x00/length 0x07/7

Mon 2016/06/27 14:48:41 gammu-smsd[3185]: gammu: 2B+|5AZ|45E|4EN|44D|0D |0A                                      +ZEND.. ```

sorry my english because I am Brazilian and I am using Google

**Comment by nijel** [comment]
What Gammu version is that and what modem do you use?

**Comment by nijel** [comment]
Should be fixed since 30faabd3 (1.37.0), #160. In case upgrading doesn't help, please include Gammu version and used modem information in the report.
